api/sys_mounts.go

```diff
@@ -242,6 +242,39 @@ func (c *Sys) MountConfigWithContext(ctx context.Context, path string) (*MountCo
 	return &result, err
 }
 
+func (c *Sys) MountInfo(path string) (*MountOutput, error) {
```

Comment:
@JanMa Interestingly, I didn't see an existing function for this. `MountConfig(...)` returns the tune information, not the mount info.